### PR TITLE
v0

### PR DESCRIPTION
**Pull request by jeangovil**

### CODE REVIEW

File: continuous-delivery.md [hunk 1]
```diff
@@ -0,0 +1,127 @@
...
+
+All environment branches should be protected; no one should be able to push commits directly to them.
+All non-protected branches are created by developers working on specific tasks and should be deleted once merged. 
+
+### Environment Branches
+You will have two environment branches on your server:
+- `main`
+- `production`
+
+Any update to the `main` branch will trigger a deployment to the staging environment, and any update to the `production` branch will trigger a deployment to the production environment. The `production` branch should never be ahead of the `main` branch.
```
[the remark this answers is not in the document]
Comment (jeangovil) [reply continuous-delivery.md:29]: As explained in the contribution rules document, these branches will be "protected," and no one will be allowed to push changes to them directly; the project's maintainer will ensure the rest.

File: development-process.md [hunk 2]
```diff
@@ -0,0 +1,51 @@
+- [Summary](#summary)
+- [Starting a New Project](#starting-a-new-project)
```
[the remark this answers is not in the document]
Comment (jeangovil) [reply development-process.md:2]: That's exactly why I explained in the document that we might take a different approach when starting a project than the one described. I suggest we wait until we do more projects of this type and find out what issues we need to address when writing such a procedure.

File: continuous-delivery.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (amirhouieh) [reply continuous-delivery.md:29]: ✅

File: development-process.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (amirhouieh) [reply development-process.md:2]: We will adjust as we go but we need to release the first version. So probably we can just skip it for now and add it in later versions

**Pull request merged**